src/tlo/methods/epilepsy.py

```diff
@@ -639,8 +662,12 @@ def apply(self, person_id, squeeze_factor):
                 priority=0
             )
 
-        else:
+        elif (
+            self._counter_of_failed_attempts_due_to_unavailable_medicines
+            < self._MAX_NUMBER_OF_FAILED_ATTEMPTS_BEFORE_DEFAULTING
+        ):
```

Comment:
(potentially highlighting gaps in my python knowledge) I would have assumed that at every instance of this HSI that self._counter_of... is set at 0 when __init__ runs so following this logic self._counter_of... would never exceed 1? Or is it the case that __init__ is only ran once? If thats the case would the counter not need to be reset to 0 incase the individual, who gives up on seeking care, develops different symptoms again and wants to seek care?

Reply:
yes, the __init__ is only called when the instance is initiated. 

When it's rescheduled, it's not created again, so __init__ is not called a second time.

Reply:
Ok perfect!